aiida/backends/sqlalchemy/utils.py

```diff
@@ -153,24 +70,25 @@ def install_tc(session):
     closure_table_parent_field = 'parent_id'
     closure_table_child_field = 'child_id'
 
-    session.execute(get_pg_tc(links_table_name, links_table_input_field,
-                              links_table_output_field, closure_table_name,
-                              closure_table_parent_field,
-                              closure_table_child_field))
+    session.execute(
+        get_pg_tc(
+            links_table_name, links_table_input_field, links_table_output_field, closure_table_name,
+            closure_table_parent_field, closure_table_child_field
+        )
+    )
 
 
-def get_pg_tc(links_table_name,
-              links_table_input_field,
-              links_table_output_field,
-              closure_table_name,
-              closure_table_parent_field,
-              closure_table_child_field):
+def get_pg_tc(
```

Comment:
Why do we still have transitive-closure code in our codebase?

Reply:
Because this is part of the original database schema and so is installed in the initial migration.